bin/bootstrap.js

```diff
@@ -31,22 +31,20 @@
 /*eslint max-statements:0, complexity:0*/
 
 // node check
-if ('process' in this && process.title === "node") {
+if ('process' in this && this.process.title === "node") {
     console.error('CasperJS cannot be executed within a nodejs environment');
-    process.exit(1);
+    this.process.exit(1);
 }
 
 // phantom check
 if (!('phantom' in this)) {
     console.error('CasperJS needs to be executed in a PhantomJS environment http://phantomjs.org/');
-} else {
-    if (phantom.version.major === 2) {
-        //setting other phantom.args if using phantomjs 2.x
-        var system = require('system');
-        var argsdeprecated = system.args;
-        argsdeprecated.shift();
-        phantom.args = argsdeprecated;
-    }
+} else if (phantom.version.major === 2) {
```

Comment:
These lines could be removed if you make [this change](https://github.com/n1k0/casperjs/pull/1310/files#diff-822825f4dd2af4a56d77d85db9fd0573R173)